Integrations/Cybereason/Cybereason.py

```diff
@@ -1029,11 +1027,23 @@ def query_file_command():
             if 'companyName' in simple_values:
                 company_name = simple_values['companyName']['values'][0]
 
+            created_time = None
+            if 'createdTime' in simple_values:
+                created_time = timestamp_to_datestring(simple_values.get('createdTime', {}).get('values')[0])
+
+            modified_time = None
+            if 'modifiedTime' in simple_values:
+                modified_time = timestamp_to_datestring(simple_values.get('modifiedTime', {}).get('values')[0])
+
+            is_signed = None
+            if 'isSigned' in simple_values:
+                is_signed = True if simple_values.get('isSigned', {}).get('values')[0] == 'true' else False
```

Comment:
This could result in an exception (if `isSigned` is not in `simple_values` and then `.get('values') ` would return None, which after, using this: `[0]` you are trying to grab the first element of `None` value)
I would suggest doing simethong like this:
```
value_list = demisto.get(simple_values, 'isSigned.values')[0] if  demisto.get(simple_values, 'isSigned.values') else 'false'
is_signed = True if value_list == 'true' else False

```